spec/features/invite_existing_user_spec.rb

```diff
@@ -12,6 +12,6 @@
     fill_in "Email", with: "existing@example.com"
     #click button: send an invitation
     find('input[name="commit"]').click
-    expect(page).to have_content "existing@example.com has been invited to join your team"
+    expect(page).to have_content "#{user.first_name} has been invited to join your team"
```

Comment:
Maybe instead writing this specific instance scenario as a spec, you create a generalized spec that contains scenarios for users and add this in.  E.g. create a file (spec/features/users_spec.rb), then put this spec in that file.  This way, when people write more specs describing UI scenarios they are all centrally located in one file.  

You could even move user_can_create_new_bill_spec.rb, user_can_sign_in_spec.rb, and invite_existing_user.rb into users_spec.rb as well.  Or, if you simply make users_spec.rb and your spec in it, I can merge the other specs regarding the user's behavior into users_spec.rb